docs/help/account/billing.md

```diff
@@ -0,0 +1,44 @@
+# Billing
+
+The Billing section of your account provides access to the following items:
+
+- information about your current plan
+- your invoice history
+- your billing details
+
+Bump.sh offers monthly or annual billing. Payment is by credit card only; we do not offer any alternatives at this time. We use Stripe to handle all payments, so we never store your bank details.
+
+## Invoices
+
+### Edit Your Billing Information
+
+You can edit the following billing information so that it appears on your upcoming invoices:
+
+- Company name
+- Address
+- VAT Number
+
+:::info
+Edited information will only appear on future invoices; it is not possible to edit the details of an already created invoice.
+:::
+
+### Invoices
+
+Every month, we send your invoice in PDF format to the email address linked to your account. You can specify an alternative email address to which they will be sent, for your accounting department, for example.
+
+On this page, you will also find a history of your invoices, available in PDF.
+
+### Taxes
+
+We apply taxes in accordance with current regulations. Three scenarios are possible:
```

Comment:
You mention three scenarios here, but we only list 2 below, I think we should update this sentence.